src/ui/components/atoms/containerContent/ContainerContent.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import ContainerContent from './ContainerContent';
+import Container from '../../../../assets/icons/Rectangle 131.svg?raw';
+
+const meta = {
+  title: 'ui/components/atoms/containerContent',
```

Comment:
Change the name of the folder by container-content, made the same with the name of the attribute title  `ui/components/atoms/container-content`